.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
FYI 

We discussing lock file in https://github.com/hexojs/hexo/issues/3370 (It hasn't reached a conclusion yet)

Reply:
For `hexo-filter-nofollow` the lock file is not necessary, since it has no dependencies at all.